src/python/grpcio/grpc/_cython/_cygrpc/aio/server.pyx.pxi

```diff
@@ -494,7 +494,7 @@ async def _handle_exceptions(RPCState rpc_state, object rpc_coro, object loop):
     except asyncio.CancelledError:
         _LOGGER.debug('RPC cancelled for servicer method [%s]', _decode(rpc_state.method()))
     except _ServerStoppedError:
-        _LOGGER.info('Aborting RPC due to server stop.')
+        _LOGGER.debug('Aborting RPC due to server stop.')
```

Comment:
Hm... Is aborting an RPC at server stop considered normal operation? This is why we have graceful shutdown internally. I think we might actually want to keep this one at info level. I'll leave it up to you though.

Reply:
If the halted RPC is considered abnormal, we should make it clearer by saying it is a `warning`. This log is only triggered if the method is actually aborted when the grace period is ended, so it might be undesired by the user. Good catch.